dags/looker.py

```diff
@@ -153,4 +153,38 @@
         **airflow_gke_prod_kwargs,
     )
 
-    lookml_generator_staging >> lookml_generator_prod
+    looker_folders_to_validate = [
+        706,  # KPI metrics
+    ]
+
+    lookml_validate_spectacles = GKEPodOperator(
+        owner="ascholtz@mozilla.com",
+        email=[
+            "ascholtz@mozilla.com",
+            "telemetry-alerts@mozilla.com",
+        ],
+        task_id="lookml_validate_spectacles",
+        name="lookml-validate-spectacles",
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest",
+        dag=dag,
+        cmds=["bash", "-x", "-c"],
+        arguments=[
+            "pip install spectacles==2.4.10 && " # todo: remove this once mozilla-nimbus-schemas supports newer pydantic version
+            "spectacles content --verbose"
+            " --base-url ${BASE_URL}"
+            " --client_id ${LOOKER_API_CLIENT_ID}"
+            " --client_secret ${LOOKER_API_CLIENT_SECRET}"
+            " --project spoke-default"
+            " --branch ${BRANCH}"
+            " --pin-imports looker-hub:main"
+            f" --folders {' --folders '.join(looker_folders_to_validate)}"
```

Comment:
The [Spectacles CLI docs](https://docs.spectacles.dev/cli/reference/content-validator/#folders) indicate `--folders` takes multiple values itself rather than being specified multiple times.
```suggestion
            f" --folders {' '.join(looker_folders_to_validate)}"
```

Reply:
Also, it looks like calling `str.join()` on `looker_folders_to_validate` fails because the values are integers and it wants strings.  You could either make the list values literal strings upfront, or do something like `str(f) for f in looker_folders_to_validate`.